Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1704,6 +1704,22 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.404,
 )
 
+upgradeWFs['PatatrackPixelOnlyTripletsAlpaka'] = PatatrackWorkflow(
+    digi = {
+        '--procModifiers': 'alpaka', # alpaka modifier activates customiseHLTForAlpaka
```

Comment:
This should not be used like this, as ```customiseHLTForAlpaka``` only works for the so-called GRun HLT menu, not any other HLT menu.

Reply:
Yes please, I was tripped up :( Appears several times in the file.